src/uu/paste/src/paste.rs

```diff
@@ -102,8 +104,10 @@ fn paste(
             None
         } else {
             let path = Path::new(&name);
-            let r = File::open(path).map_err_context(String::new)?;
-            Some(BufReader::new(r))
+            // TODO
+            // Is `map_err_context` correct here?
+            let file = File::open(path).map_err_context(String::new)?;
```

Comment:
`map_err_context` itself looks correct to me, but not the `String::new` passed to it. I think it should be `|| name` instead.